<launch>
  <rosparam command="load" file="$(find assembly_part_recognition)/param/connector_segmentation.yaml" ns="connector_segmentation" />
  <node name="connector_segmenter" pkg="assembly_part_recognition" type="connector_segmenter.py" output="screen"/>
</launch>